installer/x86_64/install-arch

```diff
@@ -57,7 +57,7 @@ init_onie_install()
     # Find target install block device.
     # - Default value from burned in install.conf
     # - Overridden with -d CLI arg
-    onie_dev="$(install_device_platform)"
+    [ -b "$onie_dev" ] || onie_dev="$(install_device_platform)"
```

Comment:
This should be an incomplete feature descripted above:

    # Find target install block device.
    # - Default value from burned in install.conf
    # - Overridden with -d CLI arg

https://github.com/opencomputeproject/onie/blob/master/installer/x86_64/install-arch#L26-L43 catches `$onie_dev` passed via `-d` argument.